### DISCUSSION TITLE
-db-path parameter explained

### DISCUSSION THREAD

**Discussion by richkent** [gilesknap/gphotos-sync#383] [Q&A]
Hi there - thanks for all your work on this.

On the outstanding google api page on the wiki here

https://gilesknap.github.io/gphotos-sync/main/explanations/googleapiissues.html?highlight=db+path#other-issues

You mention to use the -db-path. I can't see that on the gphotos-sync --help and it doesn't seem to work. Is it included - I am using 3.0.3 of gphotos-sync

I am trying to use a google drive destination using rclone as my destination path and this could be the cause of the errors I am encountering as GDrive doesn't like db locks.

Thanks

**Comment by richkent** [comment]
Sorry I am being an idiot - of course its --db-path however it would be good to have it on the --help. Thanks

**Comment by gilesknap** [comment]
hi @richkent I believe it is in the help as the help is auto generated from the command line parameters. However, it is rather hidden in the middle of a lot of options. The CLI of this project has got rather bloated over the years and probably needs a re-think.
